recipes/meta/conda_build_config.yaml

```diff
@@ -0,0 +1,9 @@
+build_variant:
+  - "qt"
+  - "osmesa"  # [linux64]
+
+python:
+  - 3.8
+  - 3.9
+  - 3.10
+  - 3.11
```

Comment:
do you absolutely need python 3.8? this is generally alot of work to maintain. I would suggest leaving users that need Python 3.8 to use the bioconda package. Please remove the `python` section.

Reply:
I removed the Python section and got a compatibility issue with ITK., so I added `- python >=3.9,<3.12`. Hopefully, it will fix it.